stob/module.c

```diff
@@ -24,23 +24,23 @@
 #include "module/instance.h"
 
 #if 0 /* unused code */
-static int level_stob_enter(struct m0_module *module);
-static void level_stob_leave(struct m0_module *module);
+M0_UNUSED static int level_stob_enter(struct m0_module *module);
+M0_UNUSED static void level_stob_leave(struct m0_module *module);
```

Comment:
All these declarations are disabled under `#if 0` - so why is this change needed?

Reply:
It is not required  but extraneous anyway

Reply:
Drop these changes if they are not required.

Reply:
Dropped